### PR TITLE
feat(api/routes) : add multiple routes

### PR DESCRIPTION
**Pull request by bancalarisantiago**
School ROutes:
- Update
- Get by id / delete by id
- model school
User:
-get by id / delete by id
- update user
- add user to school / adduser to course
Subjects:
- get by id / deelte by id

### CODE REVIEW

**Review by tomyanas — APPROVED** [review]

File: api/src/controllers/cSubject.ts [hunk 1]
```diff
@@ -24,21 +24,39 @@ export const createSubject = async (req: Request, res: Response) => {
...
+  const { id } = req.params; // req.body?
+  try{
+      const subject = await Subject.findById(id);
+      if(!subject) {
+          return res.status(404).json({msg: "Event not found"})
+      }
+      const subjectDeleted = await subject.delete();
+      res.status(200).json(subjectDeleted)
+  } catch(error) {
+      console.log(error)
```
Comment (CorregidorOscar) [inline api/src/controllers/cSubject.ts:59]: console.log

File: api/src/controllers/cUsers.ts [hunk 2]
```diff
@@ -70,8 +85,48 @@ export const createUser = async (req: Request, res: Response) => {
...
+      if(!user) {
+          return res.status(404).json({msg: "Event not found"})
+      }
+      const newUser = {
+          ...req.body
+      };
+      const userUpdated = await User.findByIdAndUpdate(id, newUser, {new : true})
+      res.status(200).json(userUpdated)
+  } catch(error) {
+      console.log(error)
```
Comment (CorregidorOscar) [inline api/src/controllers/cUsers.ts:105]: console.log

File: api/src/controllers/cUsers.ts [hunk 3]
```diff
@@ -70,8 +85,48 @@ export const createUser = async (req: Request, res: Response) => {
...
+  const { id } = req.params; // req.body?
+  try{
+      const user = await User.findById(id);
+      if(!user) {
+          return res.status(404).json({msg: "Event not found"})
+      }
+      const userDeleted = await user.delete();
+      res.status(200).json(userDeleted)
+  } catch(error) {
+      console.log(error)
```
Comment (CorregidorOscar) [inline api/src/controllers/cUsers.ts:122]: console.log

File: api/src/controllers/cSchool.ts [hunk 4]
```diff
@@ -43,3 +43,43 @@ export const createSchool = async (req: Request, res: Response) => {
...
+      }
+
+      const newSchool = {
+          ...req.body
+      };
+
+      const schoolUpdated = await School.findByIdAndUpdate(id, newSchool, {new : true})
+      res.status(200).json(schoolUpdated)
+  } catch(error) {
+      console.log(error)
```
Comment (CorregidorOscar) [inline api/src/controllers/cSchool.ts:67]: console.log

**Review by CorregidorOscar — APPROVED** [review]
quedaron unos console.log

**Pull request merged**